src/components/Dashboard/FirstSteps/index.js

```diff
@@ -50,3 +59,96 @@ export const FirstSteps = () => {
     </>
   );
 };
+
+const isFirstStepsCompleted = (systemUsageSummary) => {
+  const { hasListsCreated, hasCampaingsCreated, hasDomainsReady, hasCampaingsSent } =
+    systemUsageSummary;
+
+  return (
+    hasListsCreated === true &&
+    hasCampaingsCreated === true &&
+    hasDomainsReady === true &&
+    hasCampaingsSent === true
+  );
+};
+
+const getActionBoxInformation = (propName, completed) => {
+  switch (propName) {
+    case 'hasListsCreated':
```

Comment:
I think that this switch is making this code complex.

I think that you should make a general function `getActionBoxInformation`. Or you should create a different function for each scenario. It would be clear.